### DISCUSSION TITLE
Checking whether a type is built-in

### DISCUSSION THREAD

**Discussion by NSExceptional** [Azoy/Echo#42] [Q&A] [answered]
Is it possible to check whether a type is a built-in type without comparing it to each `KnownMetadata.Builtin`? I thought this is what the `isPOD` flag was for, but I've just learned it's not

**Comment by Azoy** [comment, marked as answer]
You can check if the type is `OpaqueMetadata` because those are the only ones who are opaque as of now. `isPOD` means "plain old datatype" so I believe things like structs, enums, and tuples, etc.
